examples/computer_vision_techniques/finding_masking_bright_pixels.py

```diff
@@ -28,7 +28,7 @@
 # then transform that pixel coordinate to a map coordinate.
 
 pixel_pos = np.argwhere(aia.data == aia.data.max()) * u.pixel
-hpc_max = aia.pixel_to_world(pixel_pos[:, 1], pixel_pos[:, 0])
+hpc_max = aia.wcs.pixel_to_world(pixel_pos[:, 1], pixel_pos[:, 0])
```

Comment:
This change should break this file, you will need to run it with `python examples/computer_vision_techniques/finding_masking_bright_pixels.py` and fix any errors.

Reply:
This will apply to every example file that was changed. They will need to be updated.